### PR TITLE
Add a blog post for Tenzir v4.9

### PR DESCRIPTION
**Pull request by dominiklohmann**
This adds a blog post for Tenzir v4.9, scheduled for tomorrow.

image

image

image

image

### CODE REVIEW

File: web/blog/tenzir-v4.9/index.md [hunk 1]
```diff
@@ -1,19 +1,85 @@
...
+
+This area chart displays the total ingress across all pipelines for the past day
+in MiB/s.
+
+```
+metrics
+| where #schema == "tenzir.metrics.operator"
+| where #import_time > 1 day ago
+| where source == true
+| where internal == false
```
Comment (mavam) [inline web/blog/tenzir-v4.9/index.md:55]:
```suggestion
| where source == true && internal == false
```

YMMV, but throwing in a conjunction can't hurt.

File: web/blog/tenzir-v4.9/index.md [hunk 2]
```diff
@@ -1,19 +1,85 @@
...
+
+The new [`chart`](/next/operators/chart) operator transforms the way you
+visualize your data on [app.tenzir.com](https://app.tenzir.com). It lets you
+depict your events graphically instead of in table form.
+
+Charting integrates seamlessly into your pipelines by simply adding the `chart`
+operator. For instance, plotting a bar chart representing the frequency of
+occurences for each protocol in `suricata.flow` events can be as simple as this:
+
+TODO: Add screenshots for all charts
```
Comment (mavam) [inline web/blog/tenzir-v4.9/index.md:28]: Stub comment to address the TODO.

**Review by mavam — APPROVED** [review]
Nothing much to add. Reads flawlessly! 💯

File: web/blog/tenzir-v4.9/index.md [hunk 3]
```diff
@@ -1,19 +1,85 @@
...
+
+```
+show partitions
+| summarize diskusage=sum(diskusage) by schema
+| python 'self.diskusage = self.diskusage / 2**30'
+| chart pie --title "Disk Usage (GiB)"
+```
+
+We're just getting started with charting! If you want to see further chart types
+added, have feedback on charting, or want to share examples of what your
```
Comment (dominiklohmann) [inline web/blog/tenzir-v4.9/index.md:73]:
```suggestion
added, have feedback on charting, or want to share examples of your
```

File: web/blog/tenzir-v4.9/index.md [hunk 4]
```diff
@@ -1,19 +1,85 @@
...
+visualizations with the chart operator, we would love to [hear from
+you](/discord).
+
+:::info Coming Soon: Dashboards
+The `chart` operator is a first step towards having dashboards directly in
+Tenzir. Any result that you see in the Explorer you will soon be able to pin and
+freely arrange on customizable dashboard.
+:::
```
Comment (jachris) [inline web/blog/tenzir-v4.9/index.md:81]:
```suggestion
:::info Coming Soon: Dashboards
The `chart` operator is a first step towards having dashboards directly in
Tenzir. Any result that you see in the Explorer you will soon be able to pin and
freely arrange on a customizable dashboard.
:::
```

File: web/blog/tenzir-v4.9/index.md [hunk 5]
```diff
@@ -46,10 +112,31 @@ export --live
 | enrich indicators --field dns.rrname
 ```
 
+The `enrich` operator gained a new `--filter` option to remove events it could
```
Comment (jachris) [inline web/blog/tenzir-v4.9/index.md:115]: Above this (can't directly comment on it because it's not part of this PR):
~~~
If you have massive amounts of indicators or a large amount of things you would
like>>>t<<< to contextualize, this feature is for you.
~~~

File: web/blog/tenzir-v4.9/index.md [hunk 6]
```diff
@@ -46,10 +112,31 @@ export --live
...
+| enrich indicators --field dns.rrname --filter
+```
+
 ## Housekeeping
 
+Other noteworthy changes and improvements:
+- `tenzir.db-directory` is now `tenzir.state-directory`. The old option remains
+  functional, but will be phased out in an upcoming release.
+- On the command-line, Tenzir now respects [`NO_COLOR`](https://no-color.org)
+  when printing diagnostics.
```
Comment (jachris) [inline web/blog/tenzir-v4.9/index.md:131]: And we do not print colors if we output device is not a terminal, making it suitable for non-interactive use, which is IMHO the more important part of the change. But you can leave as-is, I don't really mind.

File: web/blog/tenzir-v4.9/index.md [hunk 7]
```diff
@@ -46,10 +112,31 @@ export --live
...
+- RFC 5424-style Syslog parsing no longer omits structured data fields.
```
Comment (jachris) [inline web/blog/tenzir-v4.9/index.md:132]: Double negation.
```suggestion
- RFC 5424-style Syslog parsing now emits a record with the structured data fields.
```

File: web/blog/tenzir-v4.9/index.md [hunk 8]
```diff
@@ -1,19 +1,85 @@
...
 
+## Chart Operator
+
+The new [`chart`](/next/operators/chart) operator transforms the way you
+visualize your data on [app.tenzir.com](https://app.tenzir.com). It lets you
+depict your events graphically instead of in table form.
+
+Charting integrates seamlessly into your pipelines by simply adding the `chart`
+operator. For instance, plotting a bar chart representing the frequency of
+occurences for each protocol in `suricata.flow` events can be as simple as this:
```
Comment (jachris) [inline web/blog/tenzir-v4.9/index.md:26]:
```suggestion
occurrences for each protocol in `suricata.flow` events can be as simple as this:
```

File: web/blog/tenzir-v4.9/index.md (hunk 6 above)
Comment (dominiklohmann) [reply web/blog/tenzir-v4.9/index.md:131]: Added a sentence.
